lib/recurly/purchase.php

```diff
@@ -21,9 +21,10 @@ class Recurly_Purchase extends Recurly_Resource
   /**
    * Send the purchase data to the server and creates an invoice.
    *
-   * @param Recurly_Purchase Our purchase data.
-   * @param RecurlyClient Optional client for the request, useful for mocking the client
-   * @return Recurly_InvoiceCollection
+   * @param $purchase Recurly_Purchase Our purchase data.
+   * @param Recurly_Client $client Optional client for the request, useful for mocking the client
```

Comment:
If we were looking for a message to copy pasta for all these `@param Recurly_Client` lines this one is a pretty good one.

Reply:
Agreed, I'm going to change these out throughout the project

Reply:
I made them all this message, we should be good on this particular note